support-frontend/assets/pages/digital-subscription-checkout/components/digitalCheckoutForm.tsx

```diff
@@ -198,7 +199,7 @@ function DigitalCheckoutForm(props: PropTypes) {
 					<FormSection title="Address">
 						<BillingAddress countries={countries} />
 					</FormSection>
-					{paymentMethods.length > 1 ? (
+					{paymentMethods.length > 0 ? (
```

Comment:
Was this wrong before?

Reply:
Yeah I reckon this was wrong although I suppose technically before we'd never have less than 2 payment methods (we conditionally removed direct debit before but should have always had at least 2)

Reply:
Ahh I'll add a conditional check on the title that we pass to the `<FormSection>` container for 'How would you like to pay?'

Reply:
I think that will still not be right though because we'd just show the payment method selector with only one option in it and no title. What we want is to only show the payment method selector if there is more than one payment method, otherwise we should just show that payment method without the user having to select the  radio button for it.

Reply:
Ah okay I see, we want to add a new component. I'll do a bit of refactoring around this